guides/common/modules/con_salt-architecture.adoc

```diff
@@ -0,0 +1,18 @@
+[id="salt_guide_salt_architecture_{context}"]
+= Salt Architecture
+
+You need a Salt Master which either runs on your {ProjectServer} or {SmartProxyServer} with the Salt plugin enabled.
+You can also use an existing Salt Master by installing and configuring the relevant {SmartProxy} features on the existing Salt Master host.
+
+For more information on installing a Salt Master, consult the https://docs.saltproject.io/en/latest/[official Salt documentation].
+Alternatively, use the bootstrap instructions found in the https://repo.saltproject.io/[official Salt package repository].
+
+[TIP]
+====
+{Team} recommends using the same Salt version on the Salt Master as you are using on your Salt Minions.
+====
```

Comment:
Mel was very patient with many my `[NOTE]` statements.

@melcorr would you mind explaining this in the README why this is not wanted? I think it is pretty easy to avoid this, we just need to know.

Reply:
Removed `tip` and reworded `recommend`.

FYI: recommended is used a lot.